examples/nemo/eg2/runme_openmp.py

```diff
@@ -51,7 +51,7 @@
 from psyclone.parse.algorithm import parse
 from psyclone.psyir.nodes import Assignment
 from psyclone.psyGen import PSyFactory
-from psyclone.transformations import OMPParallelLoopTrans
+from psyclone.transformations import OMPParallelLoopTrans, TransformationError
```

Comment:
I'm in favour of removing this file entirely and updating the README with the instructions on how to run PSyclone directly (i.e. what the Makefile does).
What do you think?

Reply:
Agreed and done